.github/workflows/release.yaml

```diff
@@ -0,0 +1,219 @@
+name: PR Build
```

Comment:
Release? Release SPIRE? Release Tag?

Reply:
Whoopsie. Had this replaced in my test repo but forgot to port it here :)